test/l4lb/test.sh

```diff
@@ -113,3 +113,6 @@ ip r a "${LB_VIP}/32" via "$SECOND_LB_NODE_IP"
 for i in $(seq 1 10); do
     curl -o /dev/null "${LB_VIP}:80"
 done
+
+# Cleanup
+kind delete cluster
```

Comment:
Should we do the same for `test/nat46x64/test.sh`? (Not required now but in case we add more tests later on)

Reply:
It will make the workflow a bit longer though. Are we worried we will forget about deleting the previous cluster if we add more tests? If that's the case, we may mitigate the chance of this mistake happening by moving the `kind delete` to the workflow YAML's file. That way, it's right next to the new code being added, harder to miss.

Reply:
I don't see adding a new test suite in the nearest future, so I'd prefer to keep the build time shorter.